common/test/acceptance/tests/studio/test_studio_settings_details.py

```diff
@@ -65,28 +64,51 @@ def test_prerequisite_course_save_successfully(self):
         )
         pre_requisite_course_id = unicode(pre_requisite_course_key)
 
-        # refreshing the page after creating a course fixture, in order reload the pre requisite course drop down.
+        # Refresh the page to load the new course fixture and populate the prrequisite course dropdown
+        # Then select the prerequisite course and save the changes
         self.settings_detail.refresh_page()
         select_option_by_value(
             browser_query=self.settings_detail.pre_requisite_course,
             value=pre_requisite_course_id
         )
-
-        # trigger the save changes button.
         self.settings_detail.save_changes()
-
         self.assertTrue('Your changes have been saved.' in self.settings_detail.browser.page_source)
+
+        # Refresh the page again to confirm the prerequisite course selection is properly reflected
         self.settings_detail.refresh_page()
-        self.assertTrue(is_option_value_selected(browser_query=self.settings_detail.pre_requisite_course,
-                                                 value=pre_requisite_course_id))
+        self.assertTrue(is_option_value_selected(
+            browser_query=self.settings_detail.pre_requisite_course,
+            value=pre_requisite_course_id
+        ))
 
-        # now reset/update the pre requisite course to none
+        # Set the prerequisite course back to None and save the changes
         select_option_by_value(browser_query=self.settings_detail.pre_requisite_course, value='')
+        self.settings_detail.save_changes()
+        self.assertTrue('Your changes have been saved.' in self.settings_detail.browser.page_source)
```

Comment:
Do you know what element this message will be in? Looking in page source is pretty brute force.